### PR TITLE
[PLAY-433] Title kit enhancement: Bold prop/tokens

### PR DESCRIPTION
**Pull request by Israel-Molestina**
#### Screens

Screen Shot 2022-11-14 at 10 53 31 AM

Screen Shot 2022-11-14 at 10 53 37 AM

#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-433)

#### How to test this

N/A

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

File: playbook/app/pb_kits/playbook/tokens/_titles.scss [hunk 1]
```diff
@@ -31,8 +31,19 @@
 
 @mixin pb_title_4 {
   @include pb_title($heading_4, $bolder);
+  letter-spacing: -0.03em !important;
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/tokens/_titles.scss:34]: I'm worried about this important tag. We won't be able to easily override this

Comment (Israel-Molestina) [reply playbook/app/pb_kits/playbook/tokens/_titles.scss]: I removed it in my latest commit. I left it in there by accident and it is not needed. Thanks for catching that.

**Review by nidaqg — COMMENTED** [review]
Looks good!

**Comment by jasoncypret** [comment]
The changes we want.

1. We want to make `$heading_1: 46px;`
2. We want the letter spacing for this title to be `-.01em`

@Israel-Molestina

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**